### PR TITLE
Add: implement the RPMSG-tty tests

### PR DESCRIPTION
**Pull request by stanley31huang**
implement the RPMSG-tty tests

### checkbox validation:
stanley@stanley-ThinkPad-T490:~/Desktop/Git_Repos/iot/checkbox-provider-ce-oem/checkbox-provider-ce-oem$ python3 manage.py validate
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-audio-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-audio-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-button-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-button-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-button-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-button-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-buzzer-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-buzzer-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-buzzer-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-buzzer-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-cpu-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-cpu-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-cpu-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-cpu-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-accelerator-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-accelerator-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-accelerator-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-accelerator-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-crypto-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-crypto-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-crypto-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-crypto-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-dtb-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-dtb-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-dtb-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-dtb-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-digital-io-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-digital-io-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-digital-io-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-digital-io-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-eeprom-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-eeprom-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-eeprom-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-eeprom-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-installation-info-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-installation-info-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-led-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-led-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-led-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-led-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-gps-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-gps-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-gps-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-gps-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-mir-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-mir-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-mir-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-mir-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-mtd-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-mtd-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-mtd-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-mtd-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-optee-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-optee-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-otg-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-otg-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-ptp-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-ptp-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-ptp-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-ptp-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rpmsg-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rpmsg-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rtc-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rtc-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rtc-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rtc-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rs485-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rs485-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rs485-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rs485-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-socketcan-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-socketcan-stress-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-socketcan-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-socketcan-stress-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-server-22-04-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-server-22-04-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-server-20-04-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-server-20-04-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.certification::docker-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.certification::docker-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-stress
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-stress
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-ubuntucore-22-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-ubuntucore-22-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-ubuntucore-20-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-iot-ubuntucore-20-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-stress
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-stress
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-installation-info-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-cpu-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-button-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-sensor-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-gps-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-mtd-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-buzzer-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-audio-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-otg-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rtc-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rs485-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-eeprom-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-led-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-caam-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-crypto-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-touchscreen-evdev
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-socketcan-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-installation-info-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-dtb-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-cpu-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-button-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-ptp-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-sensor-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-gps-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-mtd-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-buzzer-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-audio-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-otg-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rtc-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-rs485-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-eeprom-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-led-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-accelerator-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-crypto-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-optee-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-socketcan-stress-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-cpu-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-button-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-sensor-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-gps-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-mtd-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-buzzer-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-audio-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-otg-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rtc-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rs485-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-eeprom-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-led-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-caam-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-crypto-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-touchscreen-evdev
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-socketcan-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-cpu-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-button-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-ptp-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-sensor-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-gps-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-mtd-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-buzzer-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-audio-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-otg-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rtc-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-rs485-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-eeprom-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-led-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-accelerator-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-crypto-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-optee-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-socketcan-stress-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-cold-boot-stress-test-by-pdu
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-sensor-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::ce-oem-sensor-automated
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-sensor-manual
WARNING plainbox.unit.testplan: unable to find nested part: com.canonical.qa.ceoem::after-suspend-ce-oem-sensor-automated
The provider seems to be valid

### CODE REVIEW

**Comment by stanley31huang** [comment]
I also sent a fixed for tox to run github action with correct Python version

File: checkbox-provider-ce-oem/tests/test_rpmsg_tests.py [hunk 1]
```diff
@@ -70,3 +73,48 @@ def test_arm_processor_unexpected(self):
...
+        RPMSG TTY test is not supported for Other SoC
+        """
+        with self.assertRaisesRegex(
+                SystemExit, "Unexpected CPU type."):
+            rpmsg_tests.rpmsg_tty_test_supported("mtk")
+
+    @patch("rpmsg_tests.check_rpmsg_tty_devices")
+    def test_no_rpmsg_devices(self, mock_check_rpmsg_tty_devices):
+        """
+        No RPMSG TTY devices not found and raise SystemExit
```
Comment (LiaoU3) [inline checkbox-provider-ce-oem/tests/test_rpmsg_tests.py:104]: `No RPMSG TTY devices found and raise SystemExit` or ` RPMSG TTY devices not found and raise SystemExit`, right? I prefer the last one.

**Review by LiaoU3 — CHANGES_REQUESTED** [review]
Please see my in-line comment. Thanks!

And one more thing. I saw you did some changes in `tox`. Do you think it will be better if we split this into 2 PRs and one is for the changes for `tox`?

File: checkbox-provider-ce-oem/tests/test_rpmsg_tests.py (hunk 1 above)
Comment (stanley31huang) [reply checkbox-provider-ce-oem/tests/test_rpmsg_tests.py]: fixed

**Review by LiaoU3 — APPROVED** [review]
+1

**Comment by stanley31huang** [comment]
@LiaoU3  Per our discussion, it would be better to separate the changes related to tox into another commit. The changes is landed into latest commits.

**Pull request merged**